pfcli/project.py

```diff
@@ -134,3 +143,79 @@ def delete(
     if project_id == get_current_project_id():
         project_context_path.unlink()
     typer.secho(f"Project {name} deleted.", fg=typer.colors.BLUE)
+
+
+@app.command("add-user", help="add user to project")
+def add_user(
+    name: str = typer.Argument(
+        ...,
+        help="Name of project to invite",
+    ),
+    username: str = typer.Argument(
+        ...,
+        help="username to invite",
+    ),
+    access_level: ProjectAccessLevel = typer.Argument(
+        ...,
+        help="access level of the new user",
+    )
+):
+    user_client: UserClientService = build_client(ServiceType.USER)
+
+    org_id, project_id = _check_project_and_get_id(name)
+    user_id = _get_org_user_id_by_name(org_id, username)
+
+    user_client.add_to_project(user_id, project_id, access_level)
+    typer.secho(f"User successfully added to {name}")
+
+
+@app.command("set-privilege", help="set privilege level")
+def set_privilege(
+    name: str = typer.Argument(
+        ...,
+        help="Name of project to invite",
+    ),
+    username: str = typer.Argument(
+        ...,
+        help="username to invite",
+    ),
+    access_level: ProjectAccessLevel = typer.Argument(
+        ...,
+        help="access level of the new user",
+    )
+):
+    user_client: UserClientService = build_client(ServiceType.USER)
+
+    org_id, project_id = _check_project_and_get_id(name)
+    user_id = _get_org_user_id_by_name(org_id, username)
+
+    user_client.set_project_privilege(user_id, project_id, access_level)
+    typer.secho("Privilege successfully updated")
+
+
+def _get_project_id_by_name(name: str) -> str:
+    group_project_client: GroupProjectClientService = build_client(ServiceType.GROUP_PROJECT)
+    projects = group_project_client.list_projects()
+    for project in projects:
+        if project['name'] == name:
+            return project['id']
+
+    secho_error_and_exit(f"No project exists with name {name}.")
+
+
+def _check_project_and_get_id(name: str) -> tuple[str,str]:
+    """Get org_id and project_id if valid"""
+
+    user_client: UserClientService = build_client(ServiceType.USER)
+
+    org = _get_current_org()
+    project_id = _get_project_id_by_name(name)
+
+    if org['privilege_level'] == 'owner':
```

Comment:
Better to use the defined enum

```suggestion
    if org['privilege_level'] == GroupAccessLevel.OWNER:
```